src/test/java/de/qabel/desktop/DesktopClientGuiTest.java

```diff
@@ -22,7 +24,7 @@ public void setUp() throws Exception {
     public void primaryStageShowsWhenNoAccount() throws Exception {
         startKernel(kernel);
         waitForPrimaryStage();
-        AsyncUtils.assertAsync(primaryStage::isShowing);
+        waitUntil(() -> !primaryStage.isShowing());
```

Comment:
"primaryStageShowsWhenNoAccount"
the tests name implies that the stage should be shown. No reason to assert that it is not showing